api/CharacterData.json

```
@@ -440,6 +587,55 @@
           }
         }
       },
+      "replaceWith": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/CharacterData/replaceWith",
+          "spec_url": "https://dom.spec.whatwg.org/#ref-for-dom-childnode-replacewith①",
+          "support": {
+            "chrome": {
+              "version_added": "54"
+            },
+            "chrome_android": {
+              "version_added": "54"
+            },
+            "edge": {
+              "version_added": "17"
+            },
+            "firefox": {
+              "version_added": "49"
+            },
+            "firefox_android": {
+              "version_added": "49"
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": "39"
+            },
+            "opera_android": {
+              "version_added": "41"
+            },
+            "safari": {
+              "version_added": true
```

```suggestion
              "version_added": "10"
```

Suggested by mdn-bcd-collector and confirmed by https://trac.webkit.org/changeset/186803/webkit